examples/rondpoint/src/rondpoint.idl

```diff
@@ -2,6 +2,7 @@ namespace rondpoint {
   Dictionnaire copie_dictionnaire(Dictionnaire d);
   Enumeration copie_enumeration(Enumeration e);
   sequence<Enumeration> copie_enumerations(sequence<Enumeration> e);
+  record<DOMString, Enumeration> copie_carte(record<DOMString, Enumeration> c);
```

Comment:
- IDL record type is limited: the key *has* to be a string type, however the value can be anything.
- weedle will not parse `string`, so you have to declare a `DOMString` instead (it still maps properly to a rust `String`, we actually ignore the Key type).

Reply:
I think this is an okay start, and we can figure out other key types as we go. Kinda surprised `weedle` wouldn't parse `string`, but oh well.